main/index.ts

```diff
@@ -279,6 +279,28 @@ const openProjectInIde = (project: ProjectDetails) => {
   }
 };
 
+const deleteProject = (project: ProjectDetails) => {
+  new DevonInstancesService()
+    .deleteProject(project)
+    .then((projects) => {
+      mainWindow.webContents.send('delete:project', {
+        projects: projects,
+        message: 'success',
+      });
+    })
+    .catch((error) => {
+      console.log(error);
+      mainWindow.webContents.send('delete:project', {
+        projects: [],
+        message: 'error',
+      });
+    });
+};
+
+const openProjectDirectory = (path: string) => {
+  shell.showItemInFolder(path);
+};
```

Comment:
Is this function needed? it seems to be used only once

Reply:
Yes. It's needed. This is the function to open project path location